scripts/build.sh

```diff
@@ -10,6 +10,7 @@ SOURCE="${BASH_SOURCE[0]}"
 while [ -h "$SOURCE" ] ; do SOURCE="$(readlink "$SOURCE")"; done
 DIR="$( cd -P "$( dirname "$SOURCE" )/.." && pwd )"
 
+echo $DIR
```

Comment:
Not that I'm against it, is this intended or a forgotten `echo $DIR`? 🤔

Reply:
Forgotten 😓 8bef10ec(remove echo)